packages/element-library/src/video/trim.js

```diff
@@ -30,14 +30,10 @@ import { StoryPropTypes, getTransformFlip } from '@googleforcreators/elements';
 import MediaDisplay from '../media/display';
 import { elementWithFlip } from '../shared';
 import PlayPauseButton from './playPauseButton';
-import { getBackgroundStyle, videoWithScale } from './util';
+import { getBackgroundStyle, Video as _Video } from './util';
 import Captions from './captions';
 
-const Video = styled.video`
-  position: absolute;
-  max-width: initial;
-  max-height: initial;
-  ${videoWithScale}
+const Video = styled(_Video)`
```

Comment:
We don't typically use underscore suffixes or prefixes.

We usually do `const StyledVideo = styled(Video)`

Note: when using `StyledVideo`, also need to update the `jsx-a11y/media-has-caption` config here to add it to the list:
https://github.com/GoogleForCreators/web-stories-wp/blob/5b29ef015ddc679812c0c7dcb00422a07738b520/.eslintrc#L91-L95